### DISCUSSION TITLE
Add Reserve Support

### DISCUSSION THREAD

**Discussion by creatorfromhell** [Zrips/CMI#2611] [Feature Requests]
**Description of feature/improvement request:**
Requesting support for a new economy API that added various feature enhancements such as multi-currency, async support and more. https://github.com/TheNewEconomy/Reserve/wiki/Economy-Provider 
---
### Detailed explanation of what it should do, when and in what situation: ###
It should register CMI as an economy provider for Reserve, and use the economy API when it isn't a provider. 

### Basic explanation as to why this feature is needed and/or useful to have: ###
This feature is needed to add support for the modern APIs of Reserve, and to allow plugins utilize the Reserve APIs the ability to be compatible with CMI.

**Comment by xsmeths** [comment]
I second this, it'd be nice to have the ability to use reserve with CMI
